src/Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 ﻿<Project>
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netstandard2.1;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
```

Comment:
I'm thinking we should probably add `net7.0` and `net6.0` to the list to match the latest targeting frameworks of `System.Text.Json`.